.github/workflows/build.yml

```diff
@@ -14,3 +14,9 @@ jobs:
     with:
       machine-types: "[\"ubuntu-latest\"]"
       timeout-minutes: 60
+
+  spelling:
+    name: Spelling
+    uses: Lombiq/GitHub-Actions/.github/workflows/spelling.yml@issue/NEST-365
+    with:
+      spell-check-this: Lombiq/GitHub-Actions@issue/NEST-365
```

Comment:
What does this line do exactly?

Reply:
The other two projects have their configuration in the projects themselves. This made sense because they are private projects, but this is public, so here we are using just our reusable workflow configuration from LGHA.
With this line, the project used the configuration from the issue branch in LGHA, so the few added expected words are included. Unfortunately, you have to use this line to use that issue branch configuration, changing just the workflow's branch to the issue branch won't take any effect. (more information about this [here](https://github.com/Lombiq/GitHub-Actions/blob/dev/Docs/Workflows.md#spelling-workflow) in the `spell-check-this` section)
But since now the LGHA PR is merged, you can delete the `with:` property and change the workflow's branch back to dev and it should work.